.github/workflows/tests.yaml

```diff
@@ -18,6 +18,6 @@ jobs:
         run: git submodule update --init --recursive
 
       - name: Run tests
-        run: make test-forge
+        run: make test
```

Comment:
FYI this was changed to match Goerli.
I haven't noticed that until now, but `test-forge` is just an alias to `test`.
We can discuss removing `test-forge` in the near future.

Reply:
Agreed, though I will point out that "changed to match Goerli" isn't true as it is still using 'test-forge' on Goerli.

I agree that we can make it test-forge with no consequence, though we should either do it on both the PRs ([Goerli 220](https://github.com/makerdao/spells-goerli/pull/220/files) and this one) or on neither as it would desync the repos.

I will add here as a comment that I feel like the PR title "Feature: add no-match option to make test-forge" performing the action of changing 'test-forge' to 'test' in the CI doesn't feel correct, even if the change is doing the same thing. We could always do this in a separate PR if you want?

Reply:
My bad, I was looking at the local file, but turns out I hadn't pushed it.